src/tests/ftest/control/dmg_scale.py

```diff
@@ -0,0 +1,182 @@
+        6. Pool list
+        7. Query around 80 pool metrics
+        8. Destroy all 49 pools
+        9. System stop
+        10. System start
+
+        Jira ID: DAOS-10508.
+
+        :avocado: tags=all,manual
+        :avocado: tags=deployment
+        :avocado: tags=DmgScale,test_dmg_scale
+        """
+        # This is a manual test and we need to find the durations from job.log, so add "##" to make
+        # it easy to search. The log is usually over 1 million lines.
+        self.log_step("## System query")
+        dmg_command = self.get_dmg_command()
+        dmg_command.system_query()
+
+        self.log_step("## Create a 100% pool that spans all engines")
+        pool = self.get_pool(namespace="/run/pool_1/*", create=False)
+        duration = time_pool_create(log=self.log, number=1, pool=pool)
+        self.log.info("## Single pool create duration = %.1f", duration)
+
+        self.log_step("## Pool query")
+        pool.query()
+
+        self.log_step("## Pool destroy")
+        pool.destroy()
+
+        quantity = self.params.get("quantity", "/run/pool_2_percent/*", 1)
+        msg = (f"## Create {quantity} pools spanning all the engines with each pool using a 1/50th "
+               "of the capacity")
+        self.log_step(msg)
+        pool_0 = self.get_pool(namespace="/run/pool_2_percent/*", create=False)
+        duration_0 = time_pool_create(log=self.log, number=0, pool=pool_0)
+        pools = [pool_0]
+        durations = [duration_0]
+        for count in range(1, quantity):
+            pools.append(self.get_pool(create=False))
+            # Use the SCM and NVMe size of the first pool, which uses 2% of the total size.
+            pools[-1].scm_size.update(pool_0.scm_per_rank)
+            pools[-1].nvme_size.update(pool_0.nvme_per_rank)
+            durations.append(time_pool_create(log=self.log, number=count, pool=pools[-1]))
+            msg = (f"Pool {count} created. SCM = {pools[-1].scm_per_rank}; "
+                   f"NVMe = {pools[-1].nvme_per_rank}")
+            self.log.info(msg)
+        self.log.info("## durations = %s", durations)
+        total_duration = sum(durations)
+        self.log.info("## %d pools create duration = %.1f", quantity, total_duration)
+
+        self.log_step("## Pool list")
+        dmg_command.pool_list()
+
+        self.log_step("## Query around 80 pool metrics")
+        # To save time and logs, call telemetry on the first host only. With the 80 pool metrics
+        # above, ~100K lines are printed per host.
+        telemetry_utils = TelemetryUtils(
+            dmg=dmg_command, servers=[self.server_managers[0].hosts[0]])
+        telemetry_utils.get_metrics(name=",".join(ENGINE_POOL_METRICS_SHORT))
+
+        self.log_step(f"## Destroy all {quantity} pools")
+        self.destroy_pools(pools=pools)
+
+        self.log_step("## System stop")
+        dmg_command.system_stop(force=True)
+
+        self.log_step("## System start")
+        dmg_command.system_start()
```

Comment:
Should we instead use
`self.server_managers[0].system_stop()`
and
`self.server_managers[0].system_start()`
?

Reply:
Done